core/theme.js

```diff
@@ -27,16 +27,29 @@ goog.provide('Blockly.Theme');
 
 
 /**
- * A block style or a category style.
- * @typedef {!Object.<string, string>}
- */
-Blockly.Theme.Style;
+ * A block style.
+ * @typedef {{
+  *            colourPrimary:string,
```

Comment:
You can define interfaces like this. I've split the interfaces for style into two: BlockStyle and CategoryStyle after chatting with Abby as the two accept different fields.